horovod/common/ops/adasum_gpu_operations.cc

```diff
@@ -172,33 +172,30 @@ AdasumCudaAllreduceOp::NcclHierarchical(std::vector<TensorTableEntry>& entries,
     auto nccl_result = ncclReduce(
         fused_input_data_remainder, buffer_data_remainder,
         (size_t)num_elements_remaining, GetNCCLDataType(first_entry.tensor),
-        ncclSum, root_rank, *nccl_op_context_.nccl_comm_, *cuda_op_context_.stream);
+        ncclSum, root_rank, *nccl_op_context_.nccl_comm_, *gpu_op_context_.stream);
 
     nccl_context_->ErrorCheck("ncclReduce", nccl_result);
     if (global_state_->timeline.Initialized()) {
-      cuda_context_->RecordEvent(cuda_op_context_.event_queue, NCCL_REDUCE,
-                                 *cuda_op_context_.stream);
+      gpu_context_->RecordEvent(gpu_op_context_.event_queue, NCCL_REDUCE,
+                                 *gpu_op_context_.stream);
     }
   }
 
   if (global_state_->controller->IsHomogeneous() || is_root_rank) {
-    // cudaHostAlloc is significantly slower than malloc.  Pre-allocating
+    // gpuHostAlloc is significantly slower than malloc.  Pre-allocating
```

Comment:
For consistency with other comments and NVIDIA documentation link below, this should be left as `cudaHostAlloc`.

Reply:
Done.